test/unit/react-imgix.test.jsx

```diff
@@ -41,26 +41,10 @@ const makeBackgroundWithBounds = (bounds) => (props) => (
 
 const src = "http://domain.imgix.net/image.jpg";
 let sut;
-let oldConsole, log;
-
-beforeEach(() => {
-  oldConsole = global.console;
-  delete console.log;
-  console.error = console.log;
-  log = console.log.bind(console);
-});
-afterEach(() => {
-  global.console = oldConsole;
-});
-
-function sleep(ms) {
-  return new Promise((resolve) => {
-    setTimeout(resolve, ms);
-  });
-}
+jest.spyOn(global.console, 'error').mockImplementation((error) => {console.log(error)})
```

Comment:
This is no longer mocking `console.warn`, yeah? Is this what was intended?

Reply:
No longer mocking `console.error`, don't think it was mocking `warn`? 

And yes, deleting the global console and restoring it later was causing issues in the test suites. IMO it's better practice to spy on global console. But can be convinced otherwise.